python/ray/_raylet.pyx

```diff
@@ -374,7 +375,17 @@ cdef prepare_args(
                     arg.call_site())))
 
         else:
-            serialized_arg = worker.get_serialization_context().serialize(arg)
+            try:
+                serialized_arg = worker.get_serialization_context(
+                ).serialize(arg)
+            except TypeError as e:
+                msg = (
+                    "Could not serialize the argument "
+                    f"{repr(arg)} for a task or actor "
+                    f"{function_descriptor.repr}. Check "
+                    "https://docs.ray.io/en/master/serialization.html#troubleshooting " # noqa
```

Comment:
This link looks broken

this works?
https://docs.ray.io/en/master/ray-core/objects/serialization.html

Reply:
Hmm do you mind making a PR? Will be OOO for a couple days.

I think we should have unit tests for all these links from the message...